controllers/git_tekton_resources_renovater.go

```diff
@@ -172,6 +172,7 @@ func generateConfigJS(slug string, repositories []renovateRepository, _ interfac
 				recreateClosed: true,
 				recreateWhen: "always",
 				rebaseWhen: "behind-base-branch",
+				gitIgnoredAuthors: ["rhtap-staging"],
```

Comment:
Any reason not to add the prod name as well?

Reply:
Prod one wasn't renamed

Reply:
But if there are plans to rename it as well, won't it break the same way it did in stage?

Reply:
I\m not aware of such plans. If that decided, we can add one more exception.